src/components/ArrowKeyFocusManager.js

```diff
@@ -0,0 +1,156 @@
+import _ from 'underscore';
+import {Component} from 'react';
+import PropTypes from 'prop-types';
+import * as PropTypeUtils from '../libs/PropTypeUtils';
+import CONST from '../CONST';
+import KeyboardShortcut from '../libs/KeyboardShortcut';
+
+const propTypes = {
+    /** Children to render. */
+    children: PropTypes.oneOfType([
+        PropTypes.func,
+        PropTypes.node,
+    ]).isRequired,
```

Comment:
Yeah – if we _did_ want to introduce a common set of "utility prop types", I agree this would be a great one to include in there. I think I'm going to nix that in this PR and maybe address it in a separate PR sometime in the future.